db/models.py

```diff
@@ -14,3 +14,42 @@ class Actor(models.Model):
 
     def __str__(self):
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="genres")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
Change related names, because now you can get movies in which actor is involved by `actor.genres.all()`, change it to `actor.movies.all()`, the same with genres.